src/TimePicker/TimePicker.js

```diff
@@ -178,6 +190,10 @@ TimePicker.defaultProps = {
 
 TimePicker.propDescriptions = {
     ...Time.propDescriptions,
+    localizedTextShape: {
+        meridiemAM: 'Ante meridiem for 12 hour clock.',
+        meridiemPM: 'Post meridiem for 12 hour clock.'
+    },
```

Comment:
I _think_ this is redundant.  This likely already comes from the line above where we spread the `Time.propDescriptions`.  Just try removing this and see if the values still show up in the docs.